client/jetpack-connect/remote-credentials.js

```diff
@@ -109,6 +110,9 @@ export class OrgCredentialsForm extends Component {
 	}
 
 	getError( installError ) {
+		if ( installError === 'ACTIVATION_FAILURE' || 'ACTIVATION_ON_INSTALL_FAILURE' ) {
```

Comment:
This currently catches all error cases. Should be:
`if ( installError === 'ACTIVATION_FAILURE' || installError === 'ACTIVATION_ON_INSTALL_FAILURE' )...`

Reply:
ofc! 🤦‍♀️